ql/lineup.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from django.utils.functional import cached_property
+from cached_property import cached_property

```

Comment:
A to dlaczego? Skoro jest już w Django, to nie ma potrzeby dodawania kolejnej zależności.